criu/include/magic.h

```diff
@@ -100,6 +100,7 @@
 #define BPFMAP_FILE_MAGIC    0x57506142 /* Alapayevsk */
 #define BPFMAP_DATA_MAGIC    0x64324033 /* Arkhangelsk */
 #define APPARMOR_MAGIC	     0x59423047 /* Nikolskoye */
+#define PIDFD_MAGIC	     0x12345678
```

Comment:
it should be coordinates of a town in the NNNNEEEE form

Reply:
To give an example, the coordinates of Nikolskoye (see [wikipedia](https://en.wikipedia.org/wiki/Nikolskoye,_Tosnensky_District,_Leningrad_Oblast)) are 59°42'N 30°47'E. Therefore, we use `0x59423047` as magic.

Reply:
Fixed!